gradle.properties

```diff
@@ -6,7 +6,7 @@ kotlinxSerializationVersion = 1.3.1
 exposedVersion = 0.36.2
 atomicfuVersion = 0.16.3
 grgitVersion = 4.1.1
-ktorVersion = 1.6.3
+ktorVersion = 1.6.8
```

Comment:
I tried to fix the [issue ](https://youtrack.jetbrains.com/issue/KTOR-2794) with supporting application.conf in tests. But it did not help.